jacobian_follower/src/jacobian_follower/bindings.cpp

```diff
@@ -42,5 +42,12 @@ PYBIND11_MODULE(pyjacobian_follower, m)
            py::arg("grippers"),
            py::arg("max_velocity_scaling_factor"),
            py::arg("max_acceleration_scaling_factor")
-      );
+      )
+      .def("compute_IK_solutions", &JacobianFollower::compute_IK_solutions,
+           py::arg("pose"),
+           py::arg("joint_group_name"))
+      .def("fk", &JacobianFollower::computeFK,
+       py::arg("joint_angles"),
+       py::arg("joint_group_name"))
+           ;
```

Comment:
try this formatting trick, makes it easier to add new `defs` without clang-tidy forcing the `;` on the previous line.
```suggestion
           //
           ;
```

Reply:
also it seems like these need to be reformatted, they look inconsistently indented to me

Reply:
Clang-tidy and clion both do not play nicely with this file. 
- autoformatting combines `py::arg(...)`s onto the same line
- clion annotates the `py::args(...)` with `extra` in some places but not others

![image](https://user-images.githubusercontent.com/7277089/112849790-63837680-9077-11eb-8236-4c94a36e1cc7.png)

Reply:
I would just accept the `py::arg`'s being on the same line. Readability in this code isn't super important.

Reply:
feel free to merge once everything is autoformatted, otherwise we're going to have these issues later on when someone else runs the autoformat